charts/brokencrystals/templates/_helpers.tpl

```diff
@@ -2,7 +2,7 @@
 Expand the name of the chart.
 */}}
 {{- define "brokencrystals.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 50 | trimSuffix "-" }}
```

Comment:
Why was it changed?

Reply:
I changed 63 to 50 everywhere, thinking that this would reduce the likelihood of errors due to the length of the name. Later I changed it back, it seems these parts are not actively involved.